src/services/user.js

```diff
@@ -390,17 +390,36 @@ const getLocalInfo = () => {
   }
 };
 
+function sortAtndEventsByTime(a, b) {
+
+  let timeA = a.CHDate;
+  if (a.Occurrences[0]) {
+    timeA = a.Occurrences[0][0];
+  }
+  let timeB = b.CHDate;
```

Comment:
Since these two code blocks are totally repetitive, we could possibly extract them into a different method (e.g. getAtndEventTime). That way you just call `timeA = getAtndEventTime(a)` and likewise for b. This method would be more clear and singular in it's intent, and the complexity of getting the most accurate time available is abstracted to it's own method.

Reply:
Good idea!